### PR TITLE
new tokens

### PR DESCRIPTION
**Pull request by defisaur**
https://github.com/trustwallet/assets/issues/26017

### CODE REVIEW

File: types/token.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ const (
 	STELLAR      TokenType = "STELLAR"
 	KRC20        TokenType = "KRC20"
 	AURORA       TokenType = "AURORA"
-	ALGORAND     TokenType = "ALGORAND"
```
Comment (iCell) [inline types/token.go:78]: Why are we changing this? Can you provide the references

Comment (defisaur) [reply types/token.go:78]: @iCell type should be ASA not ALGORAND https://github.com/trustwallet/assets/pull/25956#discussion_r1283776125

We have 1 asset in Cardano folder and it has ALGORAND type, I noticed what new assets Nazar trying to add have ASA type. They said all assets should have ASA type

Comment (iCell) [reply types/token.go:78]: Thanks for clarifying, please create an issue to updating the type in assets database, also preparing the SQL for it.

No rush, can be done next Monday

**Review by iCell — APPROVED** [review]

**Review by jonesk7734 — COMMENTED** [review]
Tested and working

**Review by jonesk7734 — COMMENTED** [review]
Reviewed tested and working

File: types/token.go (hunk 1 above)
Comment (defisaur) [reply types/token.go:78]: SQL executed

**Pull request merged**